packages/js/src/dashboard/components/alerts-list.js

```diff
@@ -0,0 +1,49 @@
+import { useContext } from "@wordpress/element";
+import PropTypes from "prop-types";
+import { Button } from "@yoast/ui-library";
+import { EyeOffIcon, EyeIcon } from "@heroicons/react/outline";
+import classNames from "classnames";
+import { AlertsContext } from "../routes/alert-center";
+
+/**
+ *
+ * @param {Array} items The list of items.
+ * @param {boolean} hidden Whether the items are hidden or not.
+ *
+ * @returns {JSX.Element} The list component.
+ */
+export const AlertsList = ( { items = [], hidden = false } ) => {
+	const { bulletClass } = useContext( AlertsContext );
+
+	const Eye = hidden ? EyeIcon : EyeOffIcon;
+	return (
+		<ul className="yst-mt-2">
```

Comment:
This creates more than the 24px according to the design? The padding alone is 24px.